process/process_darwin.go

```diff
@@ -389,12 +389,32 @@ func convertCPUTimes(s string) (ret float64, err error) {
 	var _tmp string
 	if strings.Contains(s, ":") {
 		_t := strings.Split(s, ":")
-		hour, err := strconv.Atoi(_t[0])
-		if err != nil {
+		switch len(_t) {
+		case 3:
+			hour, err := strconv.Atoi(_t[0])
+			if err != nil {
+				return ret, err
+			}
+			t += hour * 60 * 60 * ClockTicks
+
+			mins, err := strconv.Atoi(_t[1])
+			if err != nil {
+				return ret, err
+			}
+			t += mins * 60 * ClockTicks
+			_tmp = _t[2]
+		case 2:
+			mins, err := strconv.Atoi(_t[0])
+			if err != nil {
+				return ret, err
+			}
+			t += mins * 60 * ClockTicks
+			_tmp = _t[1]
+		case 1, 0:
+			_tmp = s
+		default:
 			return ret, err
```

Comment:
From reading the code it looks like this `err` is undefined (not explicitly defined), please use something like `return ret, fmt.Errorf("wrong cputime format")`

Reply:
Good point.

Reply:
@Lomanic  Done.